releases/README.md

```diff
@@ -6,6 +6,7 @@
 #### Ship typing information
 - MPyL can now be type checked via `mypy`.
 
+
```

Comment:
This file is automatically generated when we create a release, so no need to change it here